File: Belajar_Day_2/CustomerHelper.java
import java.util.InputMismatchException;
import java.util.Scanner;

public class CustomerHelper {
    
    public static boolean tanyaLanjut (Scanner scan) {
        System.out.print("Apakah lanjut? (y/n): ");
        String input = scan.nextLine();

        if (input.equalsIgnoreCase("y")){
            return true;
        } else if (input.equalsIgnoreCase("n")){
            return false;
        } else {
            System.out.println("Input tidak sesuai");
            return false;
        }
        
    }
    public static void cekCustomer (String nama, int umur, Double gaji){
        final int MIN_AGE = 21;
        final double MIN_SALARY_APPROVE = 8_000_000;
        final double MIN_SALARY = 3_000_000;
        String decision;

        if (umur >= MIN_AGE && gaji >= MIN_SALARY_APPROVE){
            decision = " is Eligible";
        } else if (umur >= MIN_AGE && gaji >= MIN_SALARY){
            decision = " Needs Manual Review";
        } else {
            decision = " is Rejected";
        }

        System.out.println(nama + decision);
    }

    public static void inputCustomer (String nama, int umur, Double gaji){
        Scanner scan = new Scanner(System.in);

            //input logic
        System.out.print("Masukkan nama: ");
        nama = scan.nextLine();

        if(!nama.matches("[a-zA-Z ]+")){
            throw new InputMismatchException("Name must contain letters only");
        }

        System.out.print("Masukkan umur: ");
        umur = Integer.parseInt(scan.nextLine());
        
        System.out.print("Masukkan gaji: ");
        gaji = Double.parseDouble(scan.nextLine());

        cekCustomer(nama, umur, gaji);
    }
}

File: Belajar_Day_2/CustomerHelper.py
def logic (umur, penghasilan) :
    MIN_AGE = 21
    MIN_SALARY = 3000000
    MIN_APPROVE = 8000000

    if umur >= MIN_AGE and penghasilan >= MIN_APPROVE:
        decision = "Eligible"
    elif umur >= MIN_AGE and penghasilan >= MIN_SALARY:
        decision = "Needs Manual Review"
    else :
        decision = "Not Eligible"

    #print(f"{nama} is {decision}")
    
    return decision

def display_result(nama, decision):
    print(f"{nama} is {decision}")

    
def input_cust ():
        nama = input('Masukkan nama: ')
        if not nama.replace(" ","").isalpha():
            raise Exception("Name must contain Letters Only")
        umur = int(input('Masukkan umur: '))
        penghasilan = float(input('Masukkan gaji: '))

        return nama, umur, penghasilan
        
def ask_continue ():
        lanjut = input("Apakah lanjut? (y/n): ").lower()
        if lanjut in ['y']: # contoh klo lower
            return True
        elif lanjut == 'n' or lanjut =='N': # contoh if biasa
            return False
        else :
            print("input tidak sesuai")
            return False    